docs/components/servo/_index.md

```diff
@@ -339,6 +339,14 @@ You can find additional assistance in the [Troubleshooting section](/appendix/tr
 
 {{< snippet "social.md" >}}
 
+## Test the servo
```

Comment:
This is a great start but we want this text to appear on each model page (https://docs.viam.com/components/servo/gpio/, https://docs.viam.com/components/servo/fake/, etc).

Copy pasting this content multiple times would work but then if you ever want to edit it, you'll have to edit it in all places. Therefore we put reused content into include files like [this one](https://github.com/viamrobotics/docs/pull/1678/files#diff-83a95ecfb0d79c299bf824e2e98be61fa8634ce5c2cc04abe27ca7c5a73501b4) which allow you to include the same text in multiple locations. So create a file called `servo-control.md` in the folder static/includes/components and then reference it in the pages by saying:

```
{{< readfile "/static/include/components/servo-control.md" >}}
```

In the file put the same text you added here. And then add the readfile shortcode in all pages where it is needed